src/w2grid.js

```diff
@@ -4733,6 +4733,37 @@
 
             var gridBody = $('#grid_'+ this.name +'_body', obj.box).html(bodyHTML),
                 records = $('#grid_'+ this.name +'_records', obj.box);
+            var frecords = $('#grid_'+ this.name +'_frecords', obj.box);
+            records
+            .on('mouseover', 'tr', function(ev) {
+              $('#grid_'+ this.name +'_frec_' + w2utils.escapeId($(this).attr('recid'))).addClass('w2ui-record-hover')
```

Comment:
Should `this.name` be `self.name` and var self defined above it?

Reply:
You are right, this hadn't been tested well